airflow/ui/src/components/Banner/BackfillBanner.tsx

```diff
@@ -71,12 +81,15 @@ const BackfillBanner = ({ dagId }: Props) => {
   }
 
   return (
-    <Box bg="blue.solid" color="white" fontSize="m" mr="0.5" my="1" px="2" py="1" rounded="lg">
-      <HStack>
+    <Box bg="blue.solid" borderRadius="full" color="blue.contrast" my="1" px="2" py="1">
+      <HStack alignItems="center" ml={3}>
+        <RiArrowGoBackFill />
```

Comment:
In general I like that an icon is added - but in this case it is a bit mis-leading - it feels like a "undo" acton is click-able here. Like if I want to "undo" the backfill.

Reply:
Choosing icons has been the hardest part of Airflow 3 development 🥲

Reply:
Agree. But you know the icons selected will stay for the next 5 years... so this is almost as important like adding dark mode :-D

Reply:
So treat my comment as non-blocking.